frontend/src/exams/add-exam-form-confirm.vue

```diff
@@ -117,7 +117,7 @@
 </template>
 
 <script>
-  import { mapState, mapGetters } from 'vuex'
```

Comment:
This guy shouldn't be here ☹️

Reply:
If I'm reading that diff correctly, I took a space out between s and }. I've added that space back and pushed up.